docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3'
+
+services:
+  home-assistant:
+    image: homeassistant/home-assistant
+    container_name: home-assistant
+    restart: unless-stopped
+    network_mode: host
+    ports:
```

Comment:
I suggest removing the "ports" directive, it's incompatible with `network_mode: host`.